source/includes/_queries.md

```diff
@@ -287,6 +288,58 @@ Field | Description | Constraint
 -------------- | -------------- | --------------
 Transactions | an array of transactions for given account and asset | Committed transactions
 
+## Get transactions
+
+### Purpose
+
+When an information about transaction per its transaction's hash is needed, `GetTransactions` query is sent.
```

Comment:
Sounds too complicated. Maybe rephrase to (or smth similar):
`GetTransactions serves for retrieving information about transactions by its hashes`